notification2-examples/notification2-example-client/src/main/java/c8y/example/notification/client/websocket/tootallnate/TooTallNateWebSocketClient.java

```diff
@@ -43,6 +44,8 @@ public void onOpen(ServerHandshake serverHandshake) {
 
         @Override
         public void onMessage(String message) {
+            //workaround for client issue that can start disconnection if the PONG response are delayed because of busy socket
```

Comment:
_**thought:**_ Since it is a recommendation, I would say put it in the comments. Also mention that as per WS protocol, as long as the connection has messages flowing through - it should be considered active.